Packs/FeedPlainText/Integrations/FeedPlainText/FeedPlainText.yml

```diff
@@ -40,9 +40,6 @@ configuration:
   - Email
   - URL
   - File
-  - File MD5
-  - File SHA256
-  - File SHA1
```

Comment:
What happens for customer that have these marked? can/should we somehow ignore/map them in the code?

Reply:
New server versions are mapping this to File